dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitor.kt

```diff
@@ -201,4 +187,81 @@ internal class DatadogRumMonitor(
     }
 
     // endregion
+
+    // region Internal
+
+    private fun sendCompleteView(
```

Comment:
if we are sending the events anyway even if the key is not there, I am wondering was the point to keep those keys anyway ?

Reply:
It helps in the case View B is started before you call stop for View A. When navigating from activity A to activity B, it can happen than A.onpause is called after B.onResume.
Keeping the key let us perform a check (cf line 97) to make sure that the view you're trying to stop is indeed the one active.